src/NuGet.Clients/NuGet.SolutionRestoreManager/Resources.resx

```diff
@@ -140,6 +140,9 @@
 
 Missing packages: {0}</value>
   </data>
+  <data name="PackageRefNotRestoredBecauseOfNoConsent" xml:space="preserve">
+    <value>NuGet package couldn't be restored because consent has not been granted.To give consent, open the Visual Studio Options dialog, click on the Package Manager node and check 'Allow NuGet to download missing packages during build.' You can also give consent by setting the environment variable 'EnableNuGetPackageRestore' to 'true'.</value>
```

Comment:
Seems like this message could be greatly enhanced. Please review with @anangaur 
I would think, we would say something including:
    "NuGet Restore is currently disabled. To enable it, ..."